quantum/quantum.c

```diff
@@ -312,8 +312,13 @@ bool process_record_quantum(keyrecord_t *record) {
     return false;
     case DEBUG:
       if (record->event.pressed) {
+        if (debug_enable) {
```

Comment:
You could also do this as: 
```c
debug_enable ^= 1; 
if (debug_enable) {
  print("DEBUG: enabled.\n");
} else { 
  print("DEBUG: disabled.\n");
}

Reply:
Hm, yeah. Since `print()` doesn't actually require `debug_enable` to do its thing (I was thinking it needed to be printed while it's still enabled).